src/main/twirl/uk/gov/hmrc/play/views/layouts/FooterLinks.scala.html

```diff
@@ -16,7 +16,7 @@
 
 @this()
 
-@(additionalLinks: Option[Html] = None, euExitLinks: Option[Html] = None)(implicit messages: Messages)
+@(additionalLinks: Option[Html] = None, euExitLinks: Option[Html] = None, accessibilityFooterUrl: Option[String] = None)(implicit messages: Messages)
```

Comment:
Even though this compiles fine it breaks binary compatibility https://github.com/jatcwang/binary-compatibility-guide#dont-adding-parameters-with-default-values-to-methods.  Bump major version?

Reply:
@dabd increased major version to 8 in build.sbt